recipes/PayloadHttpApiNode/README.md

```diff
@@ -0,0 +1,27 @@
+# PayloadHttpApiNode
+
+Implements a simple 'quote' API using the [payload](https://github.com/hoodunit/purescript-payload)
+HTTP backend.
```

Comment:
Since this is on line 4, it won't be included when you run `make readme`. Remove the newline and let the text wrap.

Reply:
Also, could you add something in its description that says this has similar design principles as Haskell's Servant?